package.json

```diff
@@ -8,34 +8,21 @@
     "migrate": "prisma migrate deploy && prisma db seed",
     "dev": "run-s start:docker watch",
     "clean": "run-p --print-label clean:**",
-    "test": "run-p --print-label test:**",
-    "build": "run-p --print-label build:api build:editor build:api-example build:website-example",
+    "clean:media": "docker-compose exec media rm -rf ./.media",
+    "lint": " nx run-many --target=lint --exclude website-example",
```

Comment:
now uses nx to parallelize things

website-example excluded from listing as it's the only one that fails since we never had linting on it before nx